docs/datamodel/links.rst

```diff
@@ -48,6 +48,31 @@ link.
       multi friends: Person;
     }
 
+On the other hand, backlinks work in reverse to find objects that link to the
+object, and thus assume  ``multi`` as a default. Use the ``single`` keyword 
+to declare a "to-one" backlink.
+
+.. code-block:: sdl
+    :version-lt: 3.0
+
+    type Author {
+      link posts := .<authors[is Article];
+    }
+
+    type CompanyEmployee {
+      single link company := .<employees[is Company];
+    }
+
```

Comment:
You could probably just delete this since these are identical.
```suggestion
```

Reply:
Ah, no need for the separate 3.0 version if they are the same? Okay, sounds good.